package.json

```diff
@@ -86,5 +86,6 @@
     "**/src/**/*.{tsx,jsx,ts,js}": [
       "yarn lint:write"
     ]
-  }
+  },
+  "packageManager": "yarn@1.22.22+sha1.ac34549e6aa8e7ead463a7407e1c7390f61a6610"
```

Comment:
I wonder if this is what's causing the breaks in vercel?

Reply:
Ohh, yea, whoops, let me take that out. My corepack config adds that automatically, and I usually remember not to commit it but not this time.